src/add-remove.js

```diff
@@ -0,0 +1,14 @@
+import { dataBase, updateValues, updateDatabase } from './functions.js';
+
+export function addTask(task) {
+  dataBase.push(task);
+  updateValues();
+}
```

Comment:
- [ ] It would be better if the index of the array starts from `1` instead of `0` because one of the requirements is that the property index is set to the value of the new array length. Currently, if you have 4 tasks the last task has the index of 3. Kindly fix this minor issue. 

![Capture](https://user-images.githubusercontent.com/64173189/134603341-ca2e3d22-3e3a-4c6a-9898-12d43d5b0ddc.JPG)

💪 😄